tests/test_workspace.py

```diff
@@ -286,7 +286,7 @@ def test_mets_permissions(self):
             mets_path = join(ws.directory, 'mets.xml')
             mask = umask(0)
             umask(mask)
-            assert '%o' % (stat(mets_path).st_mode & ~mask) == '100644'
+            assert (stat(mets_path).st_mode & ~mask) == 0o100664 & ~mask
```

Comment:
I think that masking on the left side should not be needed.

Reply:
Thanks! That was driving me crazy :)